Hazelcast.Net/Hazelcast.IO.Serialization/HeapData.cs

```diff
@@ -87,14 +86,19 @@ public int GetTypeId()
             {
                 return SerializationConstants.ConstantTypeNull;
             }
-            return Bits.ReadIntB(data, TypeOffset);
+            return Bits.ReadIntB(_data, TypeOffset);
         }
 
         public int GetHeapCost()
         {
             // reference (assuming compressed oops)
-            int objectRef = Bits.IntSizeInBytes;
-            return objectRef + (data != null ? ArrayHeaderSizeInBytes + data.Length : 0);
+            var objectRef = Bits.IntSizeInBytes;
+            return objectRef + (_data != null ? ArrayHeaderSizeInBytes + _data.Length : 0);
+        }
+
+        public bool IsPortable()
```

Comment:
why do we have this helper method? If we have this why not have isDataSerializable() method?